### PR TITLE
Features new

### PR DESCRIPTION
**Pull request by nour-d**

### CODE REVIEW

File: src/vetportal/AddClient.java [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
- * To change this template file, choose Tools | Templates
- * and open the template in the editor.
+/**
+ * File: AddClient.java
+ * Date: April 16, 2020
+ * @Author: Nour Debiat
+ * Purpose: This window allows the user to add new clients.
  */
 
-package vetportal;
```
Comment (BekahT) [inline src/vetportal/AddClient.java:7]: The package shouldn't be deleted.

File: src/vetportal/AuditLog.java [hunk 2]
```diff
@@ -6,7 +6,6 @@
  * Purpose: This class takes events triggered in the application and logs them
  *
  */
-package vetportal;
```
Comment (BekahT) [inline src/vetportal/AuditLog.java:9]: Package should not be deleted

File: src/vetportal/Clients.java [hunk 3]
```diff
@@ -1,13 +1,11 @@
 /**
  * File: Clients.java
- * Date: March 26, 2020
+ * Date: March 28, 2020
  * @Author: Brian Rease, Nour Debiat
  * Main POC: Brian Rease
  * Purpose: This class is used to create new Clients.
  */
 
-package vetportal;
```
Comment (BekahT) [inline src/vetportal/Clients.java]: Package should not be deleted

File: src/vetportal/DashboardsGui.java [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
- * To change this template file, choose Tools | Templates
- * and open the template in the editor.
+/**
+ * File: DashboardsGui.java
+ * Date: April 16, 2020
+ * @Author: Nour Debiat
+ * Purpose: This window displays the dashboards where users can view appointments, pets, and clients.
  */
 
-package vetportal;
```
Comment (BekahT) [inline src/vetportal/DashboardsGui.java:7]: Package should not be deleted

File: src/vetportal/DashboardsGui.java [hunk 5]
```diff
@@ -16,11 +16,11 @@
 
     //Create object for VetPortal and AddClient page:
     VetPortal vetPortal;
-    AddClient addClientPage;
+    AddClient addClient2Page;
```
Comment (BekahT) [inline src/vetportal/DashboardsGui.java]: Why is this addClient2Page instead of addClientPage?

File: src/vetportal/DashboardsGui.java [hunk 6]
```diff
@@ -231,16 +238,12 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
                         .addComponent(dashboardPanel, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
         );
 
-        addClientPage = new AddClient(vetPortal);
-        createClientBtn.addActionListener(event -> addClientPage.setVisible(true));
+        addClient2Page = new AddClient(vetPortal);
```
Comment (BekahT) [inline src/vetportal/DashboardsGui.java]: Why is this addClient2Page instead of addClientPage?

File: src/vetportal/Database.java [hunk 7]
```diff
@@ -6,8 +6,6 @@
  * Purpose: This class acts as the model for the SQLite database and contains the direct methods for specific interactions.
  */
 
-package vetportal;
```
Comment (BekahT) [inline src/vetportal/Database.java]: Package should not be deleted

File: src/vetportal/Pets.java [hunk 8]
```diff
@@ -6,8 +6,6 @@
  * Purpose: This class is used to create new Pets.
  */
 
-package vetportal;
```
Comment (BekahT) [inline src/vetportal/Pets.java]: Package should not be deleted

File: src/vetportal/Users.java [hunk 9]
```diff
@@ -6,8 +6,6 @@
  * Purpose: This class is used to create new Users.
  */
 
-package vetportal;
```
Comment (BekahT) [inline src/vetportal/Users.java]: Package should not be deleted

File: src/vetportal/VetPortal.java [hunk 10]
```diff
@@ -253,46 +259,45 @@ private void authenticateUser() {
     } //end of authenticateUser()
 
     //TODO: add follow on methods
-    public void createClient(String firstName, String lastName, String phoneNumber, String email) {
+    public Boolean createClient(JLabel warnUser, String firstName, String lastName, String phoneNumber, String email) {
 //        String firstName = "Elton";
```
Comment (BekahT) [inline src/vetportal/VetPortal.java]: If this info is no longer being used, it should be deleted. We shouldn't leave test values in the code.

File: src/vetportal/DashboardsGui.java (hunk 5 above)
Comment (nour-d) [reply src/vetportal/DashboardsGui.java]: This is either something weird from the drag and drop gui, or something Brian did. I'm not sure. Is the add Client2Page used more than addClientPage?

Comment (BekahT) [reply src/vetportal/DashboardsGui.java]: Each place where addClientPage was used addClient2Page is now being used. addClientPage is not longer used anywhere in the app. It was replaced by addClient2Page on lines 17, 239, and 240 of DashboardsGui.java 
I think we could safely revert the name to addClientPage.

Comment (nour-d) [reply src/vetportal/DashboardsGui.java]: Oh okay, that sounds fine.

**Review by BekahT — APPROVED** [review]

**Pull request merged**